src/utils/latin.ts

```diff
@@ -59,3 +59,42 @@ export function tryLatin(uint8array: Uint8Array, start: number, end: number): st
 
   return String.fromCharCode(...latinBytes);
 }
+
+/**
+ * This function is an optimization for writing small basic latin strings.
+ * @internal
+ * @remarks
+ * ### Important characteristics:
+ * - If the string length is 0 return 0, do not perform any work
+ * - If a string is longer than 25 code units return null
+ * - If any code unit exceeds 128 this function returns null
+ *
+ * @param destination - The uint8array to serialize the string to
+ * @param source - The string to turn into UTF-8 bytes if it fits in the basic latin range
+ * @param offset - The position in the destination to begin writing bytes to
+ * @returns the number of bytes written to destination if all code units are below 128, otherwise null
+ */
+export function tryWriteLatin(
```

Comment:
This feels a bit silly, but how about naming the other method `tryReadLatin` then? (Or maybe something like `tryDecodeASCII`/`tryDecodeBasicLatin` and then this one `tryEncodeASCII`/`tryEncodeBasicLatin` – these methods don't handle Latin in the sense of a ISO-8859 character set, they handle Basic Latin aka ASCII only, so why not be precise about that?) Feel free to ignore!

Reply:
I agree, happy to update the naming to be clear, might as well!

Reply:
I have been so intentional throughout these two PRs to use the term "basic latin" and not ASCII, except for in the release notes, because as you say, why not be precise 😅, so yea anyway, I agree with seeing that completely through